common/config/azure-pipelines/templates/gather-docs.yaml

```diff
@@ -69,8 +71,8 @@ steps:
     inputs:
       buildType: specific
       project: '2c48216e-e72f-48b4-a4eb-40ff1c04e8e4'
-      definition: 1734
+      definition: 7436 # iTwin.js/Docs/iTwin.js Docs - YAML
       allowPartiallySucceededBuilds: true
-      buildVersionToDownload: latestFromBranch
+      buildVersionToDownload: latest
```

Comment:
I think we'll need latestFrom branch to make it work correctly on release branches right?

Reply:
I'll fix this, we'll also need to make sure we set the `branchName` value accordingly for release branches (the default is 'refs/heads/master' so we're OK here.) I'll make sure to add a note to the docs and make add a backlog item to automate this when we create the release branch moving forward.

I did want to come back to this, I think it makes more sense to pull the .updated.json from the release pipeline and not the PR validation pipeline, but we need populate the new release pipeline with a few successful builds from the actual master and release branches first. I'd also planned on turning off triggers for pushes to a branch in the PR validation pipeline when I did this too.